.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Should we change to `weekly`? Also, this seem to only include updates to Github Actions, should we also add `pip`, so it can cover python packages?